zippy-droid/src/main/java/com/zippyid/zippydroid/wizard/WizardFragment.kt

```diff
@@ -120,6 +120,11 @@ class WizardFragment : Fragment() {
                     progressBar.visibility = View.VISIBLE
                 }
             }
+            ZippyActivity.ZippyState.RETRY -> {
+                (activity as? ZippyActivity)?.state = ZippyActivity.ZippyState.READY
```

Comment:
Atkāl būtu forši lai Wizardam ir kopīgs viewModelis ar ZippyActivity, kurš patur visus state